pkg/compression/compression.go

```diff
@@ -13,6 +13,17 @@ import (
 	"github.com/ulikunitz/xz"
 )
 
+const (
+	// GzipCompression indicates gzip compression.
+	GzipCompression = "gzip"
```

Comment:
I kind-of expected these constants to directly be `compression.Algorithm` values. (But I still need to catch up on #639 , so I’m not sure it’s completely viable.)

Reply:
Very good idea. That's changed accordingly now.